pulser-core/pulser/register/__init__.py

```diff
@@ -18,8 +18,11 @@
 from pulser.register.register3d import Register3D
 from pulser.register.register_layout import RegisterLayout
 from pulser.register.special_layouts import (
+    RectangularLatticeLayout,
     SquareLatticeLayout,
     TriangularLatticeLayout,
+    TriangularLatticeLayoutRectShape,
+    RandomLayout,
```

Comment:
These imports should be deleted as well

Reply:
Don't I need it to be able to use the new class ? Same as for the supported.py file. I am not able to import the class otherwise.

Reply:
But we agreed you would remove these classes, no?

Reply:
Oh yes my bad, I thought the command was also about the `RectangulatLatticeLayout` class.